lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -66,7 +62,7 @@ export function setFrontTokenInHeaders(res: BaseResponse, userId: string, atExpi
 }
 
 export function getCORSAllowedHeaders(): string[] {
-    return [antiCsrfHeaderKey, HEADER_RID, authorizationHeaderKey, refreshTokenHeaderKey];
+    return [antiCsrfHeaderKey, HEADER_RID, authorizationHeaderKey, refreshTokenHeaderKey, authModeHeaderKey];
```

Comment:
Do we also need to add "Authorization" to this?

Reply:
`Access-Control-Allow-Credentials` covers that.